packages/web-components/src/select/select.ts

```diff
@@ -0,0 +1,206 @@
+import { attr, FASTElement, observable } from '@microsoft/fast-element';
+import { ARIAGlobalStatesAndProperties } from '@microsoft/fast-foundation';
+import { SelectSize } from './select.options.js';
+
+/**
+ * The base class used for constructing a fluent-slider custom element
+ * @public
+ */
+export class Select extends FASTElement {
+  constructor() {
+    super();
+    this.handleChange = this.handleChange.bind(this);
+    this.handleInput = this.handleInput.bind(this);
```

Comment:
If you make these fat arrow they bind to this instance without this added overhead.